pcdet/datasets/augmentor/image_augmentor_utils.py

```diff
@@ -0,0 +1,42 @@
+import copy
+import numpy as np
+
+
+def random_flip_horizontal(image, depth_map, gt_boxes, calib):
```

Comment:
How about moving this function to augmentor_utils.py with function name like "random_image_flip_horizontal"?